modules/internal/ChapelDistribution.chpl

```diff
@@ -171,6 +171,10 @@ module ChapelDistribution {
       return ret;
     }
 
+    proc trackArrays() {
```

Comment:
nit, but inline this?

Do we chpldoc ChapelDistribution? I feel pretty confident we don't, but if we do this needs a nodoc

Reply:
I don't think we do doc for this. None of the functions here are no-doc'ed